process/smartContract/builtInFunctions/esdtTransfer.go

```diff
@@ -265,12 +265,26 @@ func checkFrozeAndPause(
 	return nil
 }
 
+func arePropertiesEmpty(properties []byte) bool {
+	for _, property := range properties {
```

Comment:
could use an early `if len(properties) > 0 { return false }`

Reply:
no. as if you do len(make([]byte, 10)) it returns 10